python/repositories.bzl

```diff
@@ -0,0 +1,14 @@
+"""This file contains macros to be called during WORKSPACE evaluation.
+
+For historic reasons, pip_repositories() is defined in //python:pip.bzl.
+"""
+
+def py_repositories():
+    """Pull in dependencies needed to use the core Python rules.
+
+    At the moment this is a placeholder hook, in that it does not actually pull
```

Comment:
This should be a normal comment, not part of the documentation.

Reply:
Done, and updated the analogous docstring in pip.bzl's `pip_repositories()`.